test/moonwall.config.json

```diff
@@ -1217,6 +1232,32 @@
                 }
             ]
         },
+        {
+            "name": "chopsticks_moonlight_upgrade",
+            "testFileDir": ["suites/rt-upgrade-chopsticks-orchestrator"],
+            "foundation": {
+                "type": "chopsticks",
+                "rtUpgradePath": "../target/release/wbuild/dancelight-runtime/dancelight_runtime.compact.compressed.wasm",
```

Comment:
same here, this is wrong

Reply:
https://github.com/moondance-labs/tanssi/pull/1063/commits/03b7caa2098fec0ea63124eeccb4cb6e23c214f4